README.rst

```diff
@@ -68,6 +98,20 @@ directly, but instead be manually inspected. E.g.:
    Note that there isn't a comma in the end of the line. The last argument is
    treated as the reason why a suggestion cannot be automatically applied.
 
+Updating
+--------
+To stay current with codespell developments it is possible to build codespell from GitHub via::
+
+    pip3 install --upgrade git+https://github.com/lucasdemarchi/codespell.git
```

Comment:
We should just say `pip` and assume this links to the correct place (people who need to use `pip3` instead of `pip` hopefully already understand their environment)